### PR TITLE
VB-3017 - Improve alerts visibility

### PR DESCRIPTION
**Pull request by adaviesMOJ**
## What does this pull request do?

Remove filtering logic of alerts on the visit-scheduler as it's now filtered in the orchestration service and used for multiple areas such as prisoner profile.

## JIRA Link
https://dsdmoj.atlassian.net/browse/VB-3017
https://dsdmoj.atlassian.net/browse/VB-4921

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitNotificationEventService.kt [hunk 1]
```diff
@@ -188,11 +183,8 @@ class VisitNotificationEventService(
   private fun processAlertsRemoved(notificationDto: PrisonerAlertCreatedUpdatedNotificationDto) {
     LOG.debug("Entered handlePrisonerAlertCreatedUpdated processAlertsRemoved")
 
-    val prisonerSupportedAlertCodes = PrisonerSupportedAlertCodeType.entries.map { it.name }.toSet()
-
-    val prisonerSupportedAlertsRemoved = notificationDto.alertsRemoved.filter { it in prisonerSupportedAlertCodes }
-    if (prisonerSupportedAlertsRemoved.isNotEmpty()) {
-      if (!notificationDto.activeAlerts.any { it in prisonerSupportedAlertCodes }) {
+    if (notificationDto.alertsRemoved.isNotEmpty()) {
+      if (notificationDto.activeAlerts.isEmpty()) {
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitNotificationEventService.kt:187]: can these be combined into an and call?

**Review by dhirajshettymoj — COMMENTED** [review]
1 minor comment.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitNotificationEventService.kt (hunk 1 above)
Comment (adaviesMOJ) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitNotificationEventService.kt:187]: yes thank you!

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Pull request merged**